cabal-install/src/Distribution/Client/Init/FileCreators.hs

```diff
@@ -80,8 +80,6 @@ writeProject (ProjectSettings opts pkgDesc libTarget exeTarget testTarget)
         when (existingCabal /= cabalContents) $
           message opts T.Warning "A .cabal file was found and not updated, if updating is desired please use the '--overwrite' option."
 
-      -- clear out last line for presentation.
-      T.putStrLn ""
```

Comment:
why? This is purely for logging in terminal output. It has nothing to do with formatting files.

Reply:
It is true that it's irrelevant to the contents of `.cabal` file. But it produces an extra empty line on the terminal (so you get two trailing empty lines), and this look completely unnecessary. So, I took liberty to remove it. In theory, this could go into a separate PR, but it feels like too much for such a simple change. Maybe a reasonable middle ground would be to put it into a separate commit, if you think it is worth it. If you firmly believe that a second trailing empty line should stay, I'm fine with reverting this change.

This is also not how all other commands behave and not how `init` used to behave: they all put one empty line, not two, in the end. E.g. `cabal-install` 3.6.2:

```
tmp/cabal-test-init/A via ❄️  impure (shell) 
❯ cabal init -n -m

Guessing dependencies...

Generating LICENSE...
Warning: unknown license type, you must put a copy in LICENSE yourself.
Generating CHANGELOG.md...
Generating app/Main.hs...
Generating A.cabal...

Warning: no synopsis given. You should edit the .cabal file and add one.
You may want to edit the .cabal file and add a Description field.

tmp/cabal-test-init/A via λ 9.0.2 via ❄️  impure (shell) 
❯ cabal build
Resolving dependencies...
Build profile: -w ghc-9.0.2 -O1
In order, the following will be built (use -v for more details):
 - A-0.1.0.0 (exe:A) (first run)
Configuring executable 'A' for A-0.1.0.0..
Preprocessing executable 'A' for A-0.1.0.0..
Building executable 'A' for A-0.1.0.0..
[1 of 1] Compiling Main             ( app/Main.hs, /home/artem/tmp/cabal-test-init/A/dist-newstyle/build/x86_64-linux/ghc-9.0.2/A-0.1.0.0/x/A/build/A/A-tmp/Main.o )
Linking /home/artem/tmp/cabal-test-init/A/dist-newstyle/build/x86_64-linux/ghc-9.0.2/A-0.1.0.0/x/A/build/A/A ...

tmp/cabal-test-init/A via λ 9.0.2 via ❄️  impure (shell) took 3s 
❯ 
```

For comparison, the current `init` with two trailing empty lines:
```
tmp/cabal-test-init/A via ❄️  impure (shell) 
❯ ~/bin/cabal-head init -n -m
Warning: this is a debug build with assertions enabled.
[Log] Guessing dependencies...
[Log] Using cabal specification: 3.6
[Warning] unknown license type, you must put a copy in LICENSE yourself.
[Log] Creating fresh file CHANGELOG.md...
[Log] Creating fresh directory ./app...
[Log] Creating fresh file app/Main.hs...
[Log] Creating fresh file A.cabal...
[Warning] No synopsis given. You should edit the .cabal file and add one.
[Info] You may want to edit the .cabal file and add a Description field.


tmp/cabal-test-init/A via λ 9.0.2 via ❄️  impure (shell) took 2s 
❯
```

Reply:
But where is that second space coming from? It seems to be a function of your terminal 

```

tmp/cabal-test-init/A via λ 9.0.2 via ❄️  impure (shell) 
```

Is not trailing output in mine: 

```
[1 of 1] Compiling Main             ( main/Main.hs, /Users/emilypillmore/haskell/cabal/dist-newstyle/build/aarch64-osx/ghc-8.10.7/cabal-install-3.9.0.0/x/cabal/build/cabal/cabal-tmp/Main.o )
Linking /Users/emilypillmore/haskell/cabal/dist-newstyle/build/aarch64-osx/ghc-8.10.7/cabal-install-3.9.0.0/x/cabal/build/cabal/cabal ...
Warning: this is a debug build with assertions enabled.
[Log] Guessing dependencies...
[Log] Using cabal specification: 3.6
[Warning] unknown license type, you must put a copy in LICENSE yourself.
[Log] Creating fresh file CHANGELOG.md...
[Log] Creating fresh directory ./app...
[Log] Creating fresh file app/Main.hs...
[Log] Creating fresh file tmp.cabal...
[Warning] No synopsis given. You should edit the .cabal file and add one.
[Info] You may want to edit the .cabal file and add a Description field.
λ П tmp → λ git t8236* → 
```

Reply:
So it seems to me that you're using a nonstandard terminal here that appends data in the form of a newline and a message about what shell you're using. I'd say that's not something I'd care to support.

Reply:
On your end, do you see that the current init leaves more trailing vertical space than init from any past release and also more than build or any other command? If yes, than why would we want having inconsistency between commands?

Reply:
(By current I mean `master`).

Reply:
cabal build: 

```
λ П tmp → λ git t8236* → cabal build
cabal: No targets given and there is no package in the current directory. Use
the target 'all' for all packages in the project or specify packages or
components by name or location. See 'cabal build --help' for more details on
target options.

λ П tmp → λ git t8236* → 
```

cabal init -n -m :

```
[1 of 1] Compiling Main             ( main/Main.hs, /Users/emilypillmore/haskell/cabal/dist-newstyle/build/aarch64-osx/ghc-8.10.7/cabal-install-3.9.0.0/x/cabal/noopt/build/cabal/cabal-tmp/Main.o )
Linking /Users/emilypillmore/haskell/cabal/dist-newstyle/build/aarch64-osx/ghc-8.10.7/cabal-install-3.9.0.0/x/cabal/noopt/build/cabal/cabal ...
Warning: this is a debug build of cabal-install with assertions enabled.
[Log] Guessing dependencies...
[Log] Using cabal specification: 3.6
[Warning] unknown license type, you must put a copy in LICENSE yourself.
[Log] Creating fresh file CHANGELOG.md...
[Log] Creating fresh directory ./app...
[Log] Creating fresh file app/Main.hs...
[Log] Creating fresh file tmp.cabal...
[Warning] No synopsis given. You should edit the .cabal file and add one.
[Info] You may want to edit the .cabal file and add a Description field.

λ П tmp → λ git master* → 
```

Reply:
It seems to me that the single-line space clearing done _after_ the command output is finished in terminal is currently standard in master. Its removal introduces inconsistency.

Reply:
Fair, thank you for experimenting. I just reverted this change.